scripts/stac/imagery/collection.py

```diff
@@ -210,94 +210,70 @@ def write_to(self, destination: str) -> None:
-          [?- Preview]
-
+          https://github.com/linz/elevation/blob/master/docs/naming.md
         Returns:
             Dataset Title
         """
         # format optional metadata
         geographic_description = self.metadata.get("geographic_description")
         historic_survey_number = self.metadata.get("historic_survey_number")
-        event = self.metadata.get("event_name")
 
         # format date for metadata
+        date = f"{self.metadata['start_datetime'].year}-{self.metadata['end_datetime'].year}"
         if self.metadata["start_datetime"].year == self.metadata["end_datetime"].year:
             date = str(self.metadata["start_datetime"].year)
-        else:
-            date = f"{self.metadata['start_datetime'].year}-{self.metadata['end_datetime'].year}"
 
         # determine dataset name
+        region = HUMAN_READABLE_REGIONS[self.metadata["region"]]
+        imagery_name = region
+        elevation_description = None
         if geographic_description:
-            name = geographic_description
-        else:
-            name = HUMAN_READABLE_REGIONS[self.metadata["region"]]
+            imagery_name = geographic_description
+            elevation_description = f"- {geographic_description}"
 
         # determine if dataset is preview
+        preview = None
         if self.metadata.get("lifecycle") == "preview":
             preview = "- Preview"
-        else:
-            preview = None
 
         if self.metadata["category"] == SCANNED_AERIAL_PHOTOS:
             if not historic_survey_number:
                 raise MissingMetadataError("historic_survey_number")
-            return " ".join(f"{name} {self.metadata['gsd']} {historic_survey_number} ({date}) {preview or ''}".split())
+            return " ".join(f"{imagery_name} {self.metadata['gsd']} {historic_survey_number} ({date}) {preview or ''}".split())
 
         if self.metadata["category"] in [
             SATELLITE_IMAGERY,
             URBAN_AERIAL_PHOTOS,
             RURAL_AERIAL_PHOTOS,
         ]:
             return " ".join(
-                f"{name} {self.metadata['gsd']} {event or ''} {DATA_CATEGORIES[self.metadata['category']]} ({date}) {preview or ''}".split()  # pylint: disable=line-too-long
+                f"{imagery_name} {self.metadata['gsd']} {DATA_CATEGORIES[self.metadata['category']]} ({date}) {preview or ''}".split()  # pylint: disable=line-too-long
             )
         if self.metadata["category"] in [DEM, DSM]:
             return " ".join(
-                f"{name} {self._elevation_title_event(event) or ''} LiDAR {self.metadata['gsd']} {DATA_CATEGORIES[self.metadata['category']]} ({date}) {preview or ''}".split()  # pylint: disable=line-too-long
+                f"{region} {elevation_description or ''} LiDAR {self.metadata['gsd']} {DATA_CATEGORIES[self.metadata['category']]} ({date}) {preview or ''}".split()  # pylint: disable=line-too-long
```

Comment:
One way to avoid really long lines and the `split`/`join` method would be to use something like this:

```python
" ".join(
    value
    for value in [
        region,
        elevation_description or None,
        "LiDAR",
        …]
    if value is not None
)

```